active/NavigationView/NavigationViewTemplateSettings/PaneTitle.md

```diff
@@ -0,0 +1,45 @@
+# Background
+PaneTitle is added to NavigationView in RS4, but it's a String. Since RS5, we introduced PaneHeader(UIElement) to deprecate PaneTitle. If both PaneTitle and PaneHeader are provided by customer, PaneHeader is used.
+
+This API update addes PaneTitle [Template-Settings](https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/template-settings-classes) to the NavigationView control.
+
+# Description
+Provides calculated PaneTitle that can be referenced as TemplatedParent sources when defining templates for a NavigationView control. Not intended for general use.

```

Comment:
```suggestion
Provides the calculated PaneTitle that should be displayed by a ControlTemplate.
This will be null if the PaneHeader property is set.
```

Reply:
```suggestion
Provides calculated PaneTitle that can be referenced as TemplatedParent sources when defining templates for a NavigationView control. Not intended for general use.

Provides the calculated PaneTitle that should be displayed by a ControlTemplate.
This will be empty if the PaneHeader property is set.


```

Reply:
```suggestion
Provides the calculated PaneTitle that should be displayed by a ControlTemplate.
This will be empty if the PaneHeader property is set.
```